src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  for (let i = 0; days > i; i++) {
+    result += 40;
+  }
+
+  if (days < 3) {
+    return result;
+  } else if (days >= 3 && days <= 6) {
```

Comment:
```suggestion
  } else if (days <= 6) {
```
you already know that days is >=3